src/Stack/PrettyPrint.hs

```diff
@@ -65,12 +68,63 @@ prettyInfo = do
 prettyWarn :: Q Exp
 prettyWarn = do
     loc <- location
-    [e| monadLoggerLog loc "" LevelWarn <=< displayWithColor . (line <>) . (warningYellow "Warning:" <+>) |]
+    [e| monadLoggerLog loc "" LevelWarn <=< displayWithColor . (line <>) . (warningColor "Warning:" <+>) . indentAfterLabel |]
 
 prettyError :: Q Exp
 prettyError = do
     loc <- location
-    [e| monadLoggerLog loc "" LevelError <=< displayWithColor . (line <>) . (errorRed "Error:" <+>) |]
+    [e| monadLoggerLog loc "" LevelError <=< displayWithColor . (line <>) . (errorColor "Error:" <+>) . indentAfterLabel |]
+
+-- TODO: Figure out how to collapse these to use the same implementation
+--       as the above ones!
+
+prettyDebugL :: Q Exp
+prettyDebugL = do
+    loc <- location
+    [e| monadLoggerLog loc "" LevelDebug <=< displayWithColor . fillSep|]
+
+prettyInfoL :: Q Exp
+prettyInfoL = do
+    loc <- location
+    [e| monadLoggerLog loc "" LevelInfo <=< displayWithColor . fillSep|]
+
+prettyWarnL :: Q Exp
+prettyWarnL = do
+    loc <- location
+    [e| monadLoggerLog loc "" LevelWarn <=< displayWithColor . (line <>) . (warningColor "Warning:" <+>) . indentAfterLabel . fillSep|]
+
+prettyErrorL :: Q Exp
+prettyErrorL = do
+    loc <- location
+    [e| monadLoggerLog loc "" LevelError <=< displayWithColor . (line <>) . (errorColor "Error:" <+>) . indentAfterLabel . fillSep|]
+
+prettyDebugS :: Q Exp
+prettyDebugS = do
+    loc <- location
+    [e| monadLoggerLog loc "" LevelDebug <=< displayWithColor . fillSep . wordDoc|]
+
+prettyInfoS :: Q Exp
+prettyInfoS = do
+    loc <- location
+    [e| monadLoggerLog loc "" LevelInfo <=< displayWithColor . fillSep . wordDoc|]
+
+prettyWarnS :: Q Exp
+prettyWarnS = do
+    loc <- location
+    [e| monadLoggerLog loc "" LevelWarn <=< displayWithColor . (line <>) . (warningColor "Warning:" <+>) . indentAfterLabel . fillSep . wordDoc|]
+
+prettyErrorS :: Q Exp
+prettyErrorS = do
+    loc <- location
+    [e| monadLoggerLog loc "" LevelError <=< displayWithColor . (line <>) . (errorColor "Error:" <+>) . indentAfterLabel . fillSep . wordDoc|]
+
+-- End of duplicates
+
+indentAfterLabel :: Doc a -> Doc a
+indentAfterLabel = align
+
+wordDoc :: String -> [Doc a]
+wordDoc = map fromString . words
```

Comment:
`wordDoc` makes me think that the argument is treated as one word.  Not sure what a better name is.  Perhaps just `flow`?  Perhaps a bit magical, but I think it should read well in the context of pretty printing.